common/static/js/src/logger.js

```diff
@@ -59,6 +59,7 @@
                 return sendRequest({
                     event_type: eventType,
                     event: JSON.stringify(data),
+                    courserun_key: typeof $$course_id !== 'undefined' ? $$course_id : null,
```

Comment:
Looks good to me-- not confident in this area to give the thumb yet. Are there other events other than logging we could implement this feature where it would hepl? (Submitting scores, etc.) Obviously if that isn't in scope all good.

Reply:
Ultimately we want to hit everything that POSTs to /event. This happens in the MFEs, which will be separate PRs (and may or may not be in scope for TNL-7752).